modules/runners/scale-up.tf

```
@@ -29,6 +29,10 @@ resource "aws_lambda_function" "scale_up" {
       RUNNERS_MAXIMUM_COUNT                = var.runners_maximum_count
       SUBNET_IDS                           = join(",", var.subnet_ids)
       ENABLE_EPHEMERAL_RUNNERS             = var.enable_ephemeral_runners
+      INSTANCE_TYPES                       = join(",", var.instance_types)
+      INSTANCE_TARGET_CAPACITY_TYPE        = var.instance_targeet_capacity_type
```

```suggestion
      INSTANCE_TARGET_CAPACITY_TYPE        = var.instance_target_capacity_type
```

This will likely cause issues elsewhere.